plugins/poll/assets/javascripts/controllers/poll-breakdown.js.es6

```diff
@@ -47,7 +47,6 @@ export default Controller.extend(ModalFunctionality, {
     loadScript("/javascripts/Chart.min.js")
       .then(() => loadScript("/javascripts/chartjs-plugin-datalabels.min.js"))
       .then(() => {
-        window.Chart.plugins.unregister(window.ChartDataLabels);
```

Comment:
This is not needed anymore in de datalabels version as it's not registered by default